server-side/AppointmentPlanner.sln

```diff
@@ -1,25 +1,25 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.31019.194
+# Visual Studio Version 17
+VisualStudioVersion = 17.5.33627.172
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AppointmentPlanner", "AppointmentPlanner\AppointmentPlanner.csproj", "{FA353395-132A-49B6-BCC0-53331607763B}"
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "AppointmentPlanner", "AppointmentPlanner\AppointmentPlanner.csproj", "{A0997EEB-5DBA-4478-8520-D6EE1481D837}"
 EndProject
 Global
```

Comment:
shall we remove solution file?

Reply:
Hi @maheshtps ,

Solution file changes reverted.

Regards,
Ravikumar Venkatesan